includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -28,23 +28,7 @@ public function getData() {
 		global $wgUser;
 
 		$data = [
-			'logo' => [
-				'header' => [
-					'type' => 'link-image',
-					'href' => $this->getHref( 'fandom-logo' ),
-					// 'image' is deprecated use 'image-data' instead
-					'image' => 'wds-company-logo-fandom-powered-by-wikia',
-					'image-data' => [
-						'type' => 'wds-svg',
-						'name' => 'wds-company-logo-fandom-powered-by-wikia',
-					],
-					'title' => [
-						'type' => 'text',
-						'value' => 'Fandom powered by Wikia'
-					],
-					'tracking_label' => 'logo',
-				]
-			],
+			'logo' => $this->getLogo(),
 			'search' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "search" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)